### PR TITLE
Doc updates for 2.0.0 release and files cleanup and restructure 

### PR DESCRIPTION
**Pull request by emdobrin**
## Description

- Doc updates for 2.0.0 registration instructions
- Use the standard format for main extension README
- Update to latest code of conduct, license year

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +34,15 @@ make init
...
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK.                 |
+| [Core extensions](https://github.com/adobe/aepsdk-core-android)                      | The Mobile Core represents the foundation of the Adobe Experience Platform mobile SDK.               |
+| [Edge Network](https://github.com/adobe/aepsdk-edge-android)                      | The Edge Network extension allows you to send data to the Adobe Edge Network from a mobile application.               |
+| [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android) | The Identity extension enable identity management from your mobile app when using the Edge Network extension. |
+| [Assurance](https://github.com/adobe/aepsdk-assurance-android)                      | The Assurance extension enables validation workflows for your SDK implementation.              |
+| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDKs.                 |
```
Comment (cacheung) [inline README.md:41]: Tim pointed out avoiding usage of the AEP acronym, so edgeIdentity changed to Adobe Experience Platform sample app for Android

File: Documentation/getting-started.md [hunk 2]
```diff
@@ -1,71 +1,97 @@
...
-		MobileCore.setLogLevel(LoggingMode.VERBOSE);
+		MobileCore.setApplication(this);
 		MobileCore.configureWithAppID(ENVIRONMENT_FILE_ID);
 
-		// register Adobe extensions
 		MobileCore.registerExtensions(
-			Arrays.asList(Consent.EXTENSION, Edge.EXTENSION, Assurance.EXTENSION),
-			o -> Log.d("MobileApp", "Mobile SDK was initialized")
+			Arrays.asList(Consent.EXTENSION, Identity.EXTENSION, Edge.EXTENSION),
+			o -> Log.d("MainApp", "AEP Mobile SDK was initialized")
```
Comment (cacheung) [inline Documentation/getting-started.md:73]: Avoiding usage of the AEP acronym, change to "Adobe Experience Platform Mobile SDK was initialized"?
 Same for Kotlin.

[the remark this answers is not in the document]
Comment (emdobrin) [reply Documentation/getting-started.md]: I kept the Android logs, and updated the text

File: README.md [hunk 3]
```diff
@@ -53,7 +34,15 @@ make init
 
 | Project                                                      | Description                                                  |
 | ------------------------------------------------------------ | ------------------------------------------------------------ |
-| [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android) | Contains Android sample app for the AEP SDK.                 |
+| [Core extensions](https://github.com/adobe/aepsdk-core-android)                      | The Mobile Core represents the foundation of the Adobe Experience Platform mobile SDK.               |
+| [Edge Network](https://github.com/adobe/aepsdk-edge-android)                      | The Edge Network extension allows you to send data to the Adobe Edge Network from a mobile application.               |
+| [Identity for Edge Network](https://github.com/adobe/aepsdk-edgeidentity-android) | The Identity extension enable identity management from your mobile app when using the Edge Network extension. |
+| [Assurance](https://github.com/adobe/aepsdk-assurance-android)                      | The Assurance extension enables validation workflows for your SDK implementation.              |
```
[the remark this answers is not in the document]
Comment (emdobrin) [reply README.md]: Updated the description

**Review by cacheung — APPROVED** [review]
Looks good!

**Pull request merged**